realm/realm-library/src/androidTest/java/io/realm/RealmTests.java

```diff
@@ -186,6 +186,31 @@ public void getInstance_writeProtectedFile() throws IOException {
     }
 
     @Test
+    public void getInstance_twiceWhenRxJavaUnavailable() {
+        // test for https://github.com/realm/realm-java/issues/2416
+
+        final RealmConfiguration configuration1 = configFactory.createConfiguration("no_RxJava.realm");
+        TestRealmConfigurationFactory.emulateRxJavaUnavailable(configuration1);
+        final RealmConfiguration configuration2 = configFactory.createConfiguration("no_RxJava.realm");
+        TestRealmConfigurationFactory.emulateRxJavaUnavailable(configuration2);
+
+        /*
```

Comment:
I would move this comment above actually getting the configurations and perhaps use `//` instead just to cut down on the boilerplate/whitespace